pkg/assembler/helpers/purl.go

```diff
@@ -25,7 +25,11 @@ import (
 	"github.com/guacsec/guac/pkg/assembler/graphql/model"
 )
 
-const PurlTypeGuac = "guac"
+const (
+	PurlTypeGuac  = "guac"
+	repoSeparator = "/"
```

Comment:
nit: i think we can skip this definition, has more context showing the const in the code.

Reply:
👍